src/arrayMethodSort.js

```diff
@@ -5,7 +5,26 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    for (let i = 0; i < this.length; i++) {
+      for (let j = 0; j < this.length - i - 1; j++) {
+        if (compareFunction && compareFunction(this[j], this[j + 1]) < 1) {
+          continue;
+        }
+
+        if (compareFunction === undefined
+            && this[j].toString() <= this[j + 1].toString()
+        ) {
+          continue;
+        }
```

Comment:
instead of this strange logic just add the default value for compareFunction

<img width="235" alt="image" src="https://github.com/mate-academy/js_array-method-sort/assets/70220667/10aa97dc-4310-49bf-a270-29db59010377">

`function(compareFunction = ...) {...}`